middleware.js

```diff
@@ -101,9 +107,7 @@ module.exports = function(options) {
 
     // This function will be called if something goes wrong
     var error = function(err, errorMessage) {
-      if (debug) {
-        log('error', 'error', errorMessage || err);
-      }
+      log('error', 'error', errorMessage || err);
```

Comment:
The severity level `error` should not be conditioned by `debug` check. Besides now the condition to check `debug` flag for severity debug in moved to the `log` function.